understanding/intro.html

```diff
@@ -79,30 +79,30 @@ <h2>Understanding the Four Principles of Accessibility</h2>
 				</li>
 			</ol>
 			<p>If any of these are not true, users with disabilities will not be able to use the Web. </p>
-			<p>Under each of the principles are guidelines and Success Criteria that help to address these principles for people with disabilities. There are many general usability guidelines that make content more usable by all people, including those with disabilities. However, in WCAG 2.1, we only include those guidelines that address problems particular to people with disabilities. This includes issues that block access or interfere with access to the Web more severely for people with disabilities.</p>
+			<p>Under each of the principles are guidelines and success criteria that help to address these principles for people with disabilities. There are many general usability guidelines that make content more usable by all people, including those with disabilities. However, in WCAG 2, we only include those guidelines that address problems particular to people with disabilities. This includes issues that block access or interfere with access to the Web more severely for people with disabilities.</p>
 		</section>
 		<section>
 			<h2>Layers of Guidance</h2>
 			<section>
 				<h3>The Guidelines</h3>
-				<p>Under each principle there is a list of guidelines that address the principle. There are a total of 12 guidelines. A convenient list of just the guidelines can be found in the <a class="gl-ref" href="https://www.w3.org/TR/WCAG21/#toc">WCAG 2.1 table of contents</a>. One of the key objectives of the guidelines is to ensure that content is directly accessible to as many people as possible, and capable of being re-presented in different forms to match different peoples' sensory, physical and cognitive abilities.</p>
+				<p>Under each principle there is a list of guidelines that address the principle. There are a total of 12 guidelines. A convenient list of just the guidelines can be found in the <a class="gl-ref" href="https://www.w3.org/TR/WCAG/#toc">WCAG 2 table of contents</a>. One of the key objectives of the guidelines is to ensure that content is directly accessible to as many people as possible, and capable of being re-presented in different forms to match different peoples' sensory, physical and cognitive abilities.</p>
```

Comment:
I was recently reading this page and I noticed that the Overview page said 13, and this page said 12. Not sure which one is correct.

```suggestion
				<p>Under each principle there is a list of guidelines that address the principle. There are a total of 13 guidelines. A convenient list of just the guidelines can be found in the <a class="gl-ref" href="https://www.w3.org/TR/WCAG/#toc">WCAG 2 table of contents</a>. One of the key objectives of the guidelines is to ensure that content is directly accessible to as many people as possible, and capable of being re-presented in different forms to match different peoples' sensory, physical and cognitive abilities.</p>
```

Reply:
actually (since wcag 2.1), 13.

Reply:
Does that "convenient list of just the guidelines" work for people? I see it if I go to the original 2.0,  https://www.w3.org/TR/2008/REC-WCAG20-20081211/#contents , but not via the one at line 88

Reply:
`#toc` links you to the table of contents, which in the more recent (2.1 and 2.2) designs is on the left-hand side (so you won't visually see yourself being taken/scrolled anywhere)